File: src/main/java/com/translation/entity/Order.java
package com.translation.entity;

import com.baomidou.mybatisplus.annotation.*;
import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@TableName("order")
public class Order {

    @TableId(type = IdType.AUTO)
    private Long id;

    private String orderNo;

    private Long userId;

    private Integer payType;

    private BigDecimal amount;

    private Integer status;

    private LocalDateTime payTime;

    private String transactionId;

    @TableLogic
    private Integer deleted;

    @TableField(fill = FieldFill.INSERT)
    private LocalDateTime createTime;

    @TableField(fill = FieldFill.INSERT_UPDATE)
    private LocalDateTime updateTime;
}

File: android/app/src/main/java/com/translation/app/domain/model/Order.kt
package com.translation.app.domain.model

import java.math.BigDecimal

data class Order(
        val id: Long, val orderNo: String, val payType: Int,
        val amount: BigDecimal, val status: Int,
        val payTime: String?, val createTime: String?, val payData: String?
)